desktop/apps/article/queries/articleBody.js

```diff
@@ -24,6 +24,21 @@ export default `
   is_super_article
   is_super_sub_article
   series_description
+  sponsor {
+    partner_dark_logo
+    partner_light_logo
+    partner_condensed_logo
+    partner_logo_link
+  }
+  seriesArticle {
+    series_description
+    sponsor {
+      partner_dark_logo
+      partner_light_logo
+      partner_condensed_logo
+      partner_logo_link
+    }
+  }
```

Comment:
We should probably make the tradeoff for an additional fetch to avoid large queries like this. But for now, this gets the `seriesArticle` if it's a series sub article, and if it's the series itself, it will just use it's own `sponsor` blob.